scipy/sparse/linalg/isolve/minres.py

```diff
@@ -85,14 +85,13 @@ def minres(A, b, x0=None, shift=0.0, tol=1e-5, maxiter=None, xtype=None,
 
     eps = finfo(xtype).eps
 
-    x = zeros(n, dtype=xtype)
 
```

Comment:
@perimosocordiae I removed the line still the CI failed. Can you tell me what is the issue?